.node-version

```diff
@@ -0,0 +1 @@
+20.11.0
```

Comment:
might i suggest `20` instead of such a specific version? cos there's nothing in the repo that needs this _exact_ version and it becomes yet another place to keep up-to-date with the latest numbers.

do note there's already https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.nvmrc

Reply:
Didn't mean to commit this - already force-pushed it out!

Reply:
i <3 force-push